onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -236,13 +239,17 @@ def clear_collected_data(self):
         self.intermediate_outputs = []
 
     def collect_data(self, data_reader: CalibrationDataReader):
+        print(self.max_intermediate_outputs)
```

Comment:
Did you intend to leave this `print` statement ?

Reply:
Oh no. Good catch!